README_EXAMPLE.md

```diff
@@ -0,0 +1,187 @@
+# Name of the repository
+A little description of what it does 
+
+
+## Setup/Configuration
+This section will cover prerequisites along with the versions for this service.
+<!-- a little summary of the bellow and or any specifics/unique -->
+
+### Prerequisites
+Ideally, a list of required tools and frameworks to run the service:
+- docker 19.03.0+
+- docker-compose v2.23.0+
+- npm 10.0.0+
+- node 20.0.0+
+<!-- please make sure that tools and framework versions are defined along with links to the installation -->
+
+### Getting started
+This should be focused on what it takes to get the service up and running along with all dependencies and env configurations
+<!-- 
+- First, ensure you're running the correct version of npm, then install dependencies using:
+```sh
+npm install
+```
+
+- Bringing up dependency services. Most of the times we would use docker e.g: 
+```sh
+docker compose up -d
+```
+
+- Database migrations if needed for the service e.g:
+```sh
+# run database migrations
+npm run db:migrate
+# followed by dabatase seeding
+npm run db:seed
+```
+
+- Any other required operations for the service to function e.g: process flows
+```sh
+# installs process flows
+npm run flows 
+
+# builds ts and tsoa
+npm run build
+```
+
+- In some cases we would need to run some scripts to be able to run service so please cover that as well e.g:`
+```sh
+# this script imports DIDs from local .env file
+./scripts/import.sh
+```
+-->
+
+
+### Development mode
+In order to run the service in development mode, please follow the instructions below.
+<!-- cover in detail what it takes to run this service locally including environment configuration -->
+- Environmental variables that are required by the service
+<!-- For example: create .env file and explain what variables are needed and where to retrieve the values, it can be a person or a tool
+Make sure that .env file contains the below variables:
+EXTERNAL_API_KEY=some-api-key
+LOCAL_USER=username
+-->
+- Ither operations required by the service to start in development mode
+- Specifics like importing storybook components
+- If we have some seeds with regard to the development mode
+- If there are multiple scenarios. e.g. environmental persona and multi-persona, please cover both in separate paragraphs
+
+## Environment variables
+This is the list of all environment variables
+<!-- A short description of environment variables, if there are any unique cases please cover and mention where to retrieve the values -->
+- Table of environemnt variables as per the below example:
```

Comment:
How are the tables generated?  Is there a tool you use/have found for doing this?

Reply:
added one -> https://www.tablesgenerator.com/markdown_tables that looks quite well developed with multiple options and other helpers